document/installation.md

```diff
@@ -0,0 +1,69 @@
+# Installation
+
+## Requirements / Dependency
+
+### System
+
+It could be working the following versions or latter, but not recommended to be used except following specific versions since it does not verify with other versions.
+
+- Ubuntu: 20.04 Focal Fossa
+- [SRT v1.4.0](https://github.com/Haivision/srt/releases/tag/v1.4.0)
+
+### Prerequisites
+
+The following packages are required to install.
+
+```bash
+apt update
+apt upgrade -y
+apt install -y g++ curl cmake pkg-config libcurl4-openssl-dev build-essential zlib1g-dev git libsrt-dev libyaml-cpp-dev
+```
+
+### Dependent Packages
+
+- [prometheus-cpp](https://github.com/jupp0r/prometheus-cpp) library
+
+  ```bash
+  cd srt-prometheus-exporter
```

Comment:
This actually comes from original doc, which was wrong repository to use here.
It should be `https://github.com/jupp0r/prometheus-cpp`.

```suggestion
  git clone https://github.com/jupp0r/prometheus-cpp
  cd prometheus-cpp
```

Reply:
addressed with https://github.com/Haivision/srt-prometheus-exporter/pull/9/commits/dee401b2e0f8c1556bacf21f8083871ff58fe4ce